### PR TITLE
Fully clean name

### PR DESCRIPTION
**Pull request by patrickbrennan-sf**

### CODE REVIEW

**Review by leanand — APPROVED** [review]

File: src/plupload.js [hunk 1]
```diff
@@ -1473,6 +1473,11 @@ plupload.Uploader = function(settings) {
 						file.name = fileName;
 					}
 					
+// 					// The below should replace all chars with potential issues from here:
```
Comment (patrickbrennan-sf) [inline src/plupload.js:1476]: This should run for all uploaded files based on the comment on 1470.

Comment (patrickbrennan-sf) [reply src/plupload.js:1476]: This is also a preemptive fix to cover potential future issues if AWS suddenly decides to enforce other bad characters.

**Review by balbinasantana — APPROVED** [review]

File: src/plupload.js (hunk 1 above)
Comment (jibrankalia-sf) [reply src/plupload.js:1476]: Would love to catch these before they hit pluploader tbh. But nice to have a final check here.

**Review by jibrankalia-sf — APPROVED** [review]

File: src/plupload.js (hunk 1 above)
Comment (patrickbrennan-sf) [reply src/plupload.js:1476]: Fair, though I feel this is a bit more dry. We won't have to add this code to every frontend that can do uploads.

**Pull request merged**